src/state/KindeProvider.tsx

```
@@ -1,256 +1,409 @@
 
-  const login = useCallback(
-    (options?: AuthOptions) => client?.login(options) || Promise.resolve(),
-    [client]
-  );
+      const optionsState: Record<string, string> = options.state || {};
 
-  const register = useCallback(
-    (options?: AuthOptions) => client?.register(options) || Promise.resolve(),
-    [client]
-  );
+      delete options.state;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)